packages/models/src/Callback.ts

```diff
@@ -13,4 +13,13 @@ export interface Callback {
   isLogicFunctionProvided: boolean
   renderArguments: string[]
   isRenderFunctionProvided: boolean
+  mockResults: CallbackResult[]
```

Comment:
Thanks!  Could you please also add a comment describing this field to the interface comments, since we have them ?

Reply:
Yea, but added comment.

Little hesitant about this.  When something is unclear it's usually a sign of bad design. Sure you can add comments to clarify in that one spot, but it would be better to fix the design.

This is why the property should have been named `mockLogicResults` so it was clear exactly what they are.